packages/notifi-react-card/lib/components/subscription/subscription-card-views/HistoryCardView.tsx

```diff
@@ -115,8 +119,20 @@ export const AlertCard: React.FC<{
           walletAddress={notification.sourceAddress ?? ''}
         />
       );
+    default:
+      // It should never come here because exception should be filtered out before. https://virtuoso.dev/troubleshooting/
+      return (
+        <AlertNotificationRow
+          handleAlertEntrySelection={handleAlertEntrySelection}
+          notificationTitle={'Unsupported notification'}
+          notificationImage={<AlertIcon icon={'INFO'} />}
+          notificationSubject={'Alert not supported yet'}
+          notificationDate={''}
```

Comment:
not super important, but we can still use `notification.createdDate` probably

Reply:
Migrated to useNotificationHistory hook with some refactor.